src/psqlgraph/util.py

```diff
@@ -70,55 +64,3 @@ def sanitize(properties):
         else:
             raise ValueError(f"Cannot serialize {type(value)} to JSONB property")
     return sanitized
-
-
-def default_backoff(retries, max_retries):
```

Comment:
Was this unused? Or redone?

Reply:
As far as I can see after much searching, it is unused.